web/annotation_editor_layer_builder.css

```diff
@@ -965,7 +997,144 @@
     }
 
     .editToolbar {
+      --editor-toolbar-colorpicker-arrow-image: url(images/editor-toolbar-colorpicker-arrow.svg);
+
       transform-origin: center !important;
+
+      .buttons {
+        .colorPicker {
+          position: relative;
+          width: auto;
+          display: flex;
+          justify-content: center;
+          align-items: center;
+          gap: 4px;
+          padding: 4px;
```

Comment:
Here, and possibly below as well, *some* of these values seem to be duplicates of ones defined further up in this file; can we perhaps reduces some unneeded duplication here?

https://github.com/mozilla/pdf.js/blob/319e85449ea6b49bb89a5542def5f88894c812ee/web/annotation_editor_layer_builder.css#L240-L243

Reply:
They aren't at the same level.